src/components/Operations.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
+import '../styles/Operations.css'
+import Deposit from '@material-ui/icons/LocalAtmSharp'
+import Payment from '@material-ui/icons/PaymentSharp'
+class Operations extends Component {
+    constructor() {
+        super()
+        this.state = {
+            amount: "",
+            vendor: "",
+            category: "",
+            changed: false
+        }
+    }
+    handleInputChange = (event) => {
+        const val = event.target.value;
+        this.setState({ [event.target.name]: val });
+    }
+    clearInputFields = () => {
+        this.setState({
+          categoryInput:"",
+          vendorInput:"",
+          amountInput:0
+        })
+      }
+    deposit = () =>{
+        if(this.state.category && this.state.vendor && this.state.amount > 0){
```

Comment:
destructuring will be helpful here https://hacks.mozilla.org/2015/05/es6-in-depth-destructuring/